src/desdemona/ui/table.cljs

```diff
@@ -2,10 +2,44 @@
   (:require [wilson.dom :as wd]
             [reagent.session :as session]))
 
+(defn toggle-key
+  "Adds/removes k in given coll and returns the coll."
+  [coll k]
+  (if (contains? coll k)
```

Comment:
Didn't we decide that this is `(some? #{k} coll)`?

Also you can just pick the fn with this and then call coll k on it, which limits repetition.